nativeJS/brad_Go/momentum/app.js

```diff
@@ -0,0 +1,33 @@
+
+const calculator = {
+  plus: function(a, b) {
+    return a + b;
+  },
+  minus: function(a, b) {
+    return a - b;
+  },
+  times: function(a, b) {
+    return a * b;
+  },
+  devide: function(a, b) {
+    return a / b;
+  },
+  modulo: function(a, b) {
+    return a % b;
+  },
+  powerof: function(a, b) {
+    return a ** b;
+  }
+};
+
+const plusResult = calculator.plus(2, 3);
```

Comment:
```suggestion
const newExam = new calculator();

const plusResult = newExam.plus(2,3);
const minusResult = newExam.minus(plusResult, 10);
const timesResult = newExam.times(10, minusResult);
const divideResult = newExam.devide(timesResult, plusResult);
const powerOfResult = newExam.powerOf(divideResult, minusResult);
```

prototype 내에서 log표시 해주기 때문에, 뒤의 console.log 줄은 필요 없겠죠?